numpy/ma/tests/test_extras.py

```diff
@@ -1039,14 +1039,14 @@ def test_empty(self):
 
         # axis 0 and 1
         b = np.ma.masked_array(np.array([], dtype=float, ndmin=2))
-        assert_equal(np.median(a, axis=0), b)
-        assert_equal(np.median(a, axis=1), b)
+        assert_equal(np.ma.median(a, axis=0), b)
```

Comment:
yes these tests are mostly copy pastes from median tests, likely forgot to update these.